plugins/ExamplePlugin/doc/Makefile

```diff
@@ -0,0 +1,21 @@
+SRCFILE = user_manual
+MANUAL_FILES = user_manual.tex
+LATEX_FLAGS=-interaction=nonstopmode
+
```

Comment:
should we link the ID of RAVEN to the plugin ID in the plugin user manual?

Reply:
I did not get it.
What do you mean?

Reply:
the RAVEN user manual has a page which list the "Document Version Information". 
Does the plugin has a similar version information?
Would it make sense to link the plugin version information to the RAVEN version information in the plugin manual?

Reply:
Yes it does.
I added it here separately because in theory the plugins are external (submodule) and they have their own version control (Git). This is the reason why the ```version.tex``` here is for the plugin only.
In this particular case, the version of the plugin is in sync with the raven repository because it is in the repo directly. But it is here just to show an example of what a plugin should contain.

Reply:
OK